src/context/MotionConfigContext.tsx

```diff
@@ -33,4 +47,6 @@ export interface MotionConfigContext {
 export const MotionConfigContext = createContext<MotionConfigContext>({
     transformPagePoint: (p) => p,
     isStatic: false,
+    documentContext: document,
+    windowContext: window,
```

Comment:
I think this will fail server-side - you could add MotionConfig to ssr.test.ts to see if it renders ok